src/client/components/Meal.js

```diff
@@ -0,0 +1,133 @@
+import React from "react";
+import { useParams } from 'react-router-dom';
+import {useState} from "react";
+import Alert from 'react-popup-alert'
+
+
+const API_URL="/api";
+
+
+function Meal({oneMeal , updateReservation, allReservations}){
+
+  let params = useParams();
+  let meal = oneMeal(parseInt(params.id));
```

Comment:
It would be better if you declared the meal using state ->
```
  const [meal, setMeal]=useState("");
```
Then, inside a useEffect hook you would call the oneMeal function and setMeal()